kornia/contrib/augmentation/sequential.py

```diff
@@ -0,0 +1,45 @@
+import torch
+import torch.nn as nn
+
+from kornia.augmentation.base import _AugmentationBase, MixAugmentationBase
+
+
+class Sequential(nn.Sequential):
+    """A sequential container for Kornia augmentation-specific functions.
+
+    Args:
+        *args (_AugmentationBase): a list of augmentation module.
+        return_transform (bool): if ``True`` return the matrix describing the transformation applied to each
+            input tensor.
+
+    Example:
+        >>> import kornia
+        >>> input = torch.randn(2, 3, 5, 6)
+        >>> aug_list = Sequential(
+        ...     kornia.augmentation.ColorJitter(0.1, 0.1, 0.1, 0.1, p=1.0),
+        ...     kornia.augmentation.RandomAffine(360, p=1.0),
+        ... return_transform=True)
+        >>> output, trans_mat = aug_list(input)
+        >>> trans_mat.shape
+        torch.Size([2, 3, 3])
+    """
+
+    def __init__(self, *args: _AugmentationBase, return_transform: bool = True) -> None:
```

Comment:
Shouldn’t we return transform all the time here? For non transform you use non.Sequential, right ?